interpreter/cling/include/cling/libc.modulemap

```diff
@@ -28,14 +28,16 @@ module "libc" [system] [extern_c] [no_undeclared_includes] {
     export *
     header "math.h"
   }
-/*  module "pthread.h" {
+  module "pthread.h" {
+	requires !header_existence
```

Comment:
Could you configure your editor to use spaces instead of tabs?

Reply:
I'm sorry I missed that earlier.